docs/commands.md

```diff
@@ -250,15 +250,15 @@
  * [**hal config provider openstack bakery edit**](#hal-config-provider-openstack-bakery-edit)
  * [**hal config provider openstack disable**](#hal-config-provider-openstack-disable)
  * [**hal config provider openstack enable**](#hal-config-provider-openstack-enable)
- * [**hal config provider oraclebmcs**](#hal-config-provider-oraclebmcs)
```

Comment:
I'm worried this kind of renaming will break any existing users... is this necessary?

Reply:
It is not strictly necessary, but it is highly desirable from our point of view.  The original name was probably an unfortunate choice, and was tied to a name that is no longer used.  So leaving it as it is could also cause possible confusion.  I would like to suffer any pain now while it is hopefully a smaller impact of changing this.

Reply:
If this change would force people who do not even use that provider to take some action, like update their hal config file, then we would definitely not want to do that.